packages/gossiplog/src/AbstractGossipLog.ts

```diff
@@ -174,12 +174,12 @@ export abstract class AbstractGossipLog<Payload = unknown> extends TypedEventEmi
 		return records.length > 0
 	}
 
-	public async get(id: string): Promise<[signature: Signature, message: Message<Payload>] | [null, null]> {
+	public async get(id: string) {
 		const record = await this.db.get<MessageRecord<Payload>>("$messages", id)
 		if (record === null) {
-			return [null, null]
+			return null
```

Comment:
Why change the get API here - is there a place where destructuring wouldn't work? It is a breaking change for downstream users and the destructuring based API seems okay to me upon a first look, so just wondering about the rationale for that.

Reply:
The goal here (see line 182) is to return `branch` in the response to `get`. Should I instead return something like `[signature, message, branch]`?